packages/frontend/src/pages/QuantifyPeriodReceiver/components/QuantifyTable.tsx

```diff
@@ -110,20 +163,55 @@ const QuantifyTable = ({ periodId, receiverId }: Props): JSX.Element | null => {
 
   return (
     <div>
-      <div className="sticky z-10 w-full p-5 space-x-4 border-t border-l border-r top-14 lg:top-0 rounded-t-xl bg-warm-gray-100 dark:bg-slate-700">
-        <MarkDuplicateButton
-          disabled={selectedPraises.length < 2}
-          onClick={(): void => setIsDuplicateDialogOpen(true)}
-        />
-        <MarkDismissedButton
-          disabled={selectedPraises.length < 1}
-          onClick={(): void => setIsDismissDialogOpen(true)}
-        />
+      <div className="flex sticky z-10 w-full p-5 border-t border-l border-r top-14 lg:top-0 rounded-t-xl bg-warm-gray-100 dark:bg-slate-700">
+        <div className="mr-4">
+          <MarkDuplicateButton
+            disabled={selectedPraises.length < 2}
+            onClick={(): void => setIsDuplicateDialogOpen(true)}
+          />
+        </div>
+
+        <div className="mr-4">
+          <MarkDismissedButton
+            disabled={selectedPraises.length < 1}
+            onClick={(): void => setIsDismissDialogOpen(true)}
+          />
+        </div>
+
+        <div>
+          <QuantifyMultipleButton
+            disabled={selectedPraises.length < 1}
+            onClick={(): void => setIsQuantifyMultipleDialogOpen(true)}
+          />
+        </div>
+
+        <div className="w-22 ml-auto h-8">
+          <SearchInput
+            handleChange={(e): void => handleSearchInput(e)}
+            placeholder="Filter"
+          />
+        </div>
       </div>
 
       <div className="overflow-x-auto rounded-t-none praise-box-wide">
         <table className="w-full table-auto">
           <tbody>
+            <tr>
+              <td colSpan={2} className="pb-4">
+                <input
+                  type="checkbox"
+                  onChange={handleToggleSelectAll}
+                  checked={selectAllChecked}
+                />
+                <span className="ml-4">Select All</span>
+              </td>
+            </tr>
+            <tr>
+              <td colSpan={5}>
+                <div className="mb-5 border-2 border-t border-warm-gray-400" />
+              </td>
+            </tr>
```

Comment:
What do  you think about moving the 'Select All' checkbox into the "toolbar" area above? That way it will be "sticky" when user scrolls down the page. Also then we could remove this additional border line. 

Something like this:
![Screenshot_2022-06-26_12-43-46](https://user-images.githubusercontent.com/159270/175825484-c2fe829c-0c17-4270-84e6-314e0b7662d3.png)